plugins/techdocs/src/components/DocsResultListItem/DocsResultListItem.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  Box,
+  Chip,
+  Divider,
+  ListItem,
+  ListItemText,
+  makeStyles,
+} from '@material-ui/core';
+import { Link } from '@backstage/core-components';
+
+const useStyles = makeStyles({
+  flexContainer: {
+    flexWrap: 'wrap',
+  },
+  itemText: {
+    width: '100%',
+    marginBottom: '1rem',
+  },
+});
+
+export const DocsResultListItem = ({ result }: any) => {
+  const classes = useStyles();
+  if (result.location.includes('#')) return null;
```

Comment:
I absolutely believe that we should handle this differently and not return these results in the first place on the global search side. I just don't like the idea of creating a dependency from search functionality to what ever the types on these are to implement custom logic on search. We should probably add some interface alongside collators/decorators that could be implemented to enrich the search query itself in cases like this. 
For now it's probably better to keep this as 'best effort'.

Reply:
Not sure why you filter these out, aren't these the locations that point to an exact heading in techdocs? I find it quite useful that clicking on a search result jumps me directly into the part of the docs site that is releveant.

Reply:
^^ See discussion above regarding this

Reply:
Hmm, I think I like your approach of showing the subitems with a breadcrumb path better than hiding them in the global search as well. @iamEAP any thoughts on this?

Reply:
Yeah +1 on not filtering out here and +1 on a breadcrumb display!

When I mentioned filtering them out in the global display, I was referring to providing a custom `translator` in `packages/backend/src/plugins/search.ts`.

Reply:
Using the translator field on the SearchEngine seems a bit like nuclear option here but I don't really see other ways in current APIs to approach this without introducing dependency from plugins to specific engine nor introducing dependency from engine to specific plugins. 

The only available engine and its translator seems to still be in flux so I'll leave that be for now. It's better for userspace to have a cleaner version of it to look example from. 

To creep the scope a bit more I will add frontend pagination to the example search page though. React is starting to shows some signs of hiccup with a bit larger dataset.